client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -92,8 +96,24 @@ const DesignSelector: React.FunctionComponent = () => {
 								/>
 							</span>
 							<span className="design-selector__option-overlay">
-								<span id={ makeOptionId( design ) } className="design-selector__option-name">
-									{ design.title }
+								<span id={ makeOptionId( design ) } className="design-selector__option-meta">
+									<span className="design-selector__option-name">{ design.title }</span>
+									{ design.is_premium && (
+										<Tooltip
+											position="bottom center"
+											text={ __( 'Requires a Personal plan or above' ) }
+										>
+											<div className="design-selector__premium-container">
+												<Badge className="design-selector__premium-badge">
+													<JetpackLogo
+														className="design-selector__premium-badge-logo"
+														size={ 20 }
+													/>
+													{ __( 'Premium' ) }
```

Comment:
Optional: there needs to be in a span for the `> * { vertical-align: center }` [from above](https://github.com/Automattic/wp-calypso/pull/42377/files/7e62a2b81da9c893f099625a8b50e40520e89544#diff-b192a5e77b3fbc1b51bcdddc5f8dfd27R9-R11) to apply.